lib/queue/queue.py

```diff
@@ -98,27 +118,29 @@ def get_or_create_queues(self, queue_name: str) -> List[boto3.resources.base.Ser
             else:
                 raise
 
-    def get_sqs(self) -> boto3.resources.base.ServiceResource:
+    def create_queue(self, queue_name: str):
         """
-        Get an instantiated SQS - if local, use local alternative via elasticmq
+        Create a queue by name.
         """
-        deploy_env = get_environment_setting("DEPLOY_ENV")
-        if deploy_env == "local":
-            logger.info(f"Using ElasticMQ Interface")
-            with self.sqs_lock:
-                return boto3.resource('sqs',
-                                      region_name=(get_environment_setting("AWS_DEFAULT_REGION") or 'eu-central-1'),
-                                      endpoint_url=(get_environment_setting("ELASTICMQ_URI") or 'http://presto-elasticmq:9324'),
-                                      aws_access_key_id=(get_environment_setting("AWS_ACCESS_KEY_ID") or 'x'),
-                                      aws_secret_access_key=(get_environment_setting("AWS_SECRET_ACCESS_KEY") or 'x'))
-        else:
-            logger.info(f"Using SQS Interface")
-            with self.sqs_lock:
-                return boto3.resource('sqs', region_name=get_environment_setting("AWS_DEFAULT_REGION"))
+        attributes = {}
+        if queue_name.endswith('.fifo'):
+            attributes['FifoQueue'] = 'true'
+            attributes['ContentBasedDeduplication'] = 'true'
```

Comment:
Is attributes['ContentBasedDeduplication'] related to the queue ending in .fifo or is it something we want all the time?

Reply:
its a fifo thing - and I realized I copypasta'd here twice so removing one of these!

Reply:
Right; the deduplication is an aspect of how FIFO queues work (where you avoid processing the same message more than once). I don't think we'll ever have a scenario with FIFO, but without deduplication, but it doesn't seem unreasonable to separate these aspects.